docs/jasmine-upgrade.md

```diff
@@ -0,0 +1,76 @@
+## Upgrading from Jasmine 1.3 to 2.0
+
+First, please read the official upgrade document: http://jasmine.github.io/2.0/upgrading.html
+
```

Comment:
Add a note about how minijasminenode provided asynchronous support before, so the upgrade isn't as large as it is from unmodified Jasmine 1.3 to Jasmine 2.0